docs/usage/node-local-dns.md

```diff
@@ -51,8 +51,19 @@ For more information about `node-local-dns` please refer to the [KEP](https://gi
 ## Known Issues
 
 The [custom DNS configuration](custom-dns-config.md) may not work as expected in conjunction with `NodeLocalDNS`.
-With `NodeLocalDNS`, ordinary dns queries targetted at the upstream DNS servers, i.e. non-kubernetes domains,
+With `NodeLocalDNS`, ordinary dns queries targeted at the upstream DNS servers, i.e. non-kubernetes domains,
 will not end up at CoreDNS, but will instead be directly sent to the upstream DNS server. Therefore, configuration
 applying to non-kubernetes entities, e.g. the `istio.server` block in the
 [custom DNS configuration](custom-dns-config.md) example, may not have any effect with `NodeLocalDNS` enabled.
-If this kind of custom configuration is required `NodeLocalDNS` needs to be disabled.
+If this kind of custom configuration is required, forwarding to upstream DNS has to be disabled.
```

Comment:
Why do we have duplicated the same documentation? Can you decide for only once place and add a link to it from the other one?